app/assets/javascripts/quby/answers/validation.js

```diff
@@ -126,7 +126,7 @@
                   if(value === undefined || value == ""){
                       continue;
                   }
-                  var rgx = /(\s*-?[1-9]+[0-9]*\s*|\s*-?[0-9]?\s*)/;
+                  var rgx = /(\s*-?[1-9]+[0-9]+\s*|\s*-?[0-9]\s*)/;
```

Comment:
Als ik nu `01` invul capturet hij `0`. De check hieronder (`result != value`) faalt dan.

Als er nog vragenlijsten zijn waar integervragen worden gebruikt om een geboortedag te vragen, lijkt het me niet onwaarschijnlijk dat mensen 01 invullen.

Misschien is het dus een redelijk idee om alle integerwaarden toe te staan, ook als ze met een 0 beginnen. Maakt de regex direct ook simpeler (`(\s*-?[0-9]*\s*)` dacht ik aan)